docs/extensions/tutorials/_posts/1970-01-01-PublishYourApp.md

```diff
@@ -224,25 +215,15 @@ This is a manual process of publishing an android app in the Google Play Store.
 
 ### Building your app
 
-If you have all the requirements met, let's build your app. The result of this step is an unsigned build file (.apk file), which you can use to submit your app to the store.
-
-To do this, use the Shoutem CLI command:
-
-```ShellSession
-$ shoutem build-android
-  Select your app: Restaurants ({{ site.example.appId }})
-  ...
-```
-
-The build file will be located in the current folder.
+To prepare your app for release, you will have to use `shoutem configure --release` in the cloned app's directory. After doing this short step, the build process is functionally identical to a [normal React Native app](https://facebook.github.io/react-native/docs/signed-apk-android.html).
 
 ### Preparing for Store
 
 Once you have the build ready, you can start preparing your app for store.
 
 #### App Signing and Zipalign
 
-You need to sign the build file (.apk) that you got from the Shoutem CLI. You need to sign your android apk with your own keystore. Here's the [official documentation](https://developer.android.com/studio/publish/app-signing.html) on how to do that.
+If you generated an unsigned build (.apk) you need to sign it with your own keystore. Here's the [official documentation](https://developer.android.com/studio/publish/app-signing.html) on how to do that.
```

Comment:
I think it's not "if", but rather "once you generated an unsigned build", right?